src/ui/flutter_app/lib/screens/game_page/game_page.dart

```diff
@@ -524,26 +383,24 @@ class _GamePageState extends State<GamePage>
         gameInstance.isAiToMove &&
         mounted) {
       if (widget.engineType == EngineType.aiVsAi) {
-        final String score =
```

Comment:
Although it affects performance, this way of writing can improve readability. The code is written for humans first and then for the machine.